### DISCUSSION TITLE
Seeing Preview Env values on Production

### DISCUSSION THREAD

**Discussion by piesrtasty** [vercel/community#932] [Help]
I'm at my wit's end. I am seeing preview env vars in production. When I refer to "production" I mean any of the following:

1. any code deployed from the "main" branch
2. any deployment explicitly "promoted to production"
3. deploying with the Vercel CLI with "vercel --prod"

I'm losing my mind at this point. There must be something I'm missing?

**Comment by amyegan** [comment]
Hi @lukehamilton. Sorry you're running into this problem. Can you share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) or example deployment to help me get a better look at what is going wrong?
